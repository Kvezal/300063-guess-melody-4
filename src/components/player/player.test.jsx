import React from "react";
import render from "react-test-renderer";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Player from "./player";


Enzyme.configure({
  adapter: new Adapter(),
});

describe(`PlayerComponent`, () => {
  it(`should render component`, () => {
    const tree = render.create(<Player />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should contain audio`, () => {
    const player = shallow(<Player />);
    const audio = player.find(`audio`);
    expect(audio).toHaveLength(1);
  });
});
