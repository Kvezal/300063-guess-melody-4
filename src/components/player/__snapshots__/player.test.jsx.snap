// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerComponent should render component 1`] = `
<div
  className="track__status"
>
  <audio />
</div>
`;
